src/Targets/Microsoft.CSharp.DesignTime.targets

```diff
@@ -26,6 +26,13 @@
     <PropertyPageSchema Include="$(CustomCommonXamlResourcesDirectory)AppDesigner.xaml">
       <Context>ProjectSubscriptionService</Context>
     </PropertyPageSchema>
+  
+    <PropertyPageSchema Include="$(CustomCommonXamlResourcesDirectory)General.BrowseObject.xaml">
+      <Context>BrowseObject</Context>
+    </PropertyPageSchema>
+  
+    <PropertyPageSchema Include="$(CustomCommonXamlResourcesDirectory)None.xaml">
+      <Context>File;BrowseObject</Context>
```

Comment:
The Context tells CPS the services to which these properties should be published. All the item related ones are published to File as well. I don't know how to get at them yet.